blueprints/bridge-deployments-wireguard.html.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Bridge your other deployments to Fly.io
+layout: docs
+sitemap: true
+nav: firecracker
+author: xe
+categories:
+  - networking
+date: 2024-05-27
+---
+
+Sometimes you can recreate production on Fly.io without any issues. Other times you need to be able to incrementally move things over. Fly.io [has private networking](https://fly.io/docs/networking/private-networking/), but something you may not realize is that you can easily connect your existing servers to your private network. This lets you use your private network as a way to incrementally move services over.
```

Comment:
Suggestion to elucidate private network and external server a bit and take out the "you may not realize" bit. Take or leave or modify this suggestion at will.
```suggestion
Sometimes you can recreate production on Fly.io without any issues. Other times you need to be able to incrementally move things over. Fly.io [has private networking](https://fly.io/docs/networking/private-networking/) by default for apps in the same organization, but you can also easily connect your existing external servers to this private network. This lets you use your private network as a way to incrementally move services over.
```